tfdf-penguins/index.html

```diff
@@ -0,0 +1,181 @@
+<!-- Copyright 2023 Google LLC. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+============================================================================== -->
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Yggdrasil Decision Forests in TensorFlow.JS</title>
+
+  <!-- Import @tensorflow/tfjs -->
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script>
+
+  <!-- Import @tensorflow/tfjs-tfdf
+  Note that we need to explicitly load dist/tf-tfdf.min.js so that it can
+  locate WASM module files from their default location (dist/). -->
+  <!-- TODO: Make TFDF search for WASM path relative to ./dist/ instead of ./ -->
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-tfdf/dist/tf-tfdf.min.js"></script>
+
+  <!-- Import papaparse to parse the penguins csv file -->
+  <script src="https://cdn.jsdelivr.net/npm/papaparse@5.4.0/papaparse.min.js"></script>
+
+  <style>
+    .button_box {
+      display: flex;
+      justify-content: space-around;
+    }
+
+    .button_box button {
+      width: 100%;
```

Comment:
You could consider setting the width to 200px (or whatever), rather than 100%. I didn't realize that this was a button at first, because -- wide! :-)